inferenceservice.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    openshift.io/display-name: granite-3.1-8b-lab-v1-1.4.0 - v1
+    serving.knative.openshift.io/enablePassthrough: "true"
+    serving.kserve.io/deploymentMode: Serverless
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+  labels:
+    modelregistry.opendatahub.io/inference-service-id: "4"
```

Comment:
these labels are from the fact that the model registry controller is involved in the deploying of the model and it help allows for the mapping of kubeflow and kserve data that the bridge merges into one definition of a running model

I left these in here @thepetk to help illustrate that deploying a model via gitops could get tricky when considering how the bridge works

@johnmcollier FYI

Reply:
the operator that links the kserve metadata with the kubeflow metadata, like most operators, expects to be the owner/creator/maintainer of the CR instance @thepetk @johnmcollier

Reply:
Ooh, that could get problematic no? If the controller updates the CR with some label/annotation/etc, the controller could presumably get stuck with ArgoCD overwriting it (and it trying to add it back)

though with this presumably just being for dev / test purposes, I wonder if we really need auto sync 🤔